api/pom.xml

```diff
@@ -69,6 +69,13 @@
 			<artifactId>event-api</artifactId>
 			<type>test-jar</type>
 		</dependency>
+
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-all</artifactId>
```

Comment:
I had thought that this usually is provided via openmrs-core. @rkorytkowski can you advise on what the best-practice is for this? (Actually, it would be great to write the answer on a wiki page and point to it.)